backend/handlers/handlers.go

```diff
@@ -639,20 +685,37 @@ func (handlers *Handlers) getConvertToPlainFiatHandler(r *http.Request) (interfa
 	from := r.URL.Query().Get("from")
 	to := r.URL.Query().Get("to")
 	amount := r.URL.Query().Get("amount")
-	amountRat, valid := new(big.Rat).SetString(amount)
-	if !valid {
+
+	currentCoin, err := handlers.backend.Coin(coinpkg.Code(strings.ToLower(from)))
```

Comment:
i don't think `from` is a reliable field for the coin code. The frontend send `coinUnit` in for this field, which might or might not be the coincode. 

Why not just provide `coinCode` as a GET parameter? Then it is 100% correct, without the heuristic of trying to convert the unit to the coin code.

Reply:
Good point, thx! Now should be better